docs/en/stack/ml/nlp/ml-nlp-autoscaling.asciidoc

```diff
@@ -0,0 +1,144 @@
+[[ml-nlp-auto-scale]]
+= Trained model autoscaling
+
+You can enable autoscaling for each of your trained model deployments.
+Autoscaling allows {es} to automatically adjust the resources the deployment can use based on the workload demand.
+
+There are two ways to enable autoscaling:
+
+* trough APIs by enabling adaptive allocations
+* in {kib} by enabling adaptive resources
+
+IMPORTANT: You must enable {cloud}/ec-autoscaling.html[deployment autoscaling] to use model autoscaling.
```

Comment:
This is not entirely true. You can just allocate one ML node and use adaptive allocations. Adaptive allocations will allocate more or less of the ML node depending on the load. However, if adaptive allocations decides it needs more than the available resources (the ML node), it cannot scale to that.

Adaptive allocations certainly works better in combination with ML autoscaling.

I'd make this a (strong) recommendation, but not say that you must enable it, since that's not true.